.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+jobs:
+  build:
+    docker:
+      - image: uxpin2/node8:1.0
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+    working_directory: ~/project/
+    steps:
+      - checkout
+      - run:
+          name: "Install dependencies"
+          command: npm install
+      - run:
+          name: "Run tests"
+          command: npm run test
+      - run:
+          name: "Check ESLint"
+          command: ./node_modules/eslint/bin/eslint.js --ext js,jsx src --ignore-pattern *.tests.js
```

Comment:
it could be run as `npm run check`

Reply:
You mean - moving this whole command to package.json and firing it here? Cause there's no `npm run check` task yet